config/katello.migrations/161216113043-move_puppet_to_top_level.rb

```diff
@@ -0,0 +1,18 @@
+answers[:puppet] ||= {
+                       :server => true,
+                       :server_environments_owner => 'apache',
+                       :server_foreman_ssl_cert => '/etc/foreman/client_cert.pem',
+                       :server_foreman_ssl_key => '/etc/foreman/client_key.pem',
+                       :server_foreman_ssl_ca => '/etc/foreman/proxy_ca.pem'
+                     }
+
+scenario[:order] = [
+  'certs',
+  'foreman',
+  'katello',
+  'foreman_proxy',
+  'foreman_proxy::plugin::pulp',
+  'foreman_proxy_content',
+  'certs::puppet',
+  'puppet'
+]
```

Comment:
> If anything else changes the order

Well if anything else changes the order it should do the migration in the same way which would be run after this one.

In this case, I could just append it I guess, but not all cases of altering the order can just be a simple append.